p2p/server.go

```diff
@@ -830,6 +835,10 @@ func (srv *Server) postHandshakeChecks(peers map[enode.ID]*Peer, inboundCount in
 	case c.node.ID() == srv.localnode.ID():
 		return DiscSelf
 	default:
+		if !c.is(trustedConn) && discfilter.Banned(c.node.ID(), c.node.Record()) && rand.Intn(5) != 0 {
+			// rarely accept useless peers
+			return DiscTooManyPeers
```

Comment:
Should we have a dedicated error for "useless" peers?

Reply:
Maybe... but we may want to hide this reason from the peer